jenkins/model_output_processing_Jenkinsfile

```diff
@@ -90,21 +90,33 @@ pipeline {
       always {
           script {
             env.DATE = sh(script: 'cat date.txt', , returnStdout: true).trim()
+            if (env.TIER == 'beta') {
+                        targetDomain = "https://labs-beta.waterdata.usgs.gov"
+            }
+            if (env.TIER == 'prod') {
+                        targetDomain = "https://labs.waterdata.usgs.gov"
+            }
+            if (env.TIER == 'test' ) {
+                        targetDomain = "https://wbeep-test-website.s3-us-west-2.amazonaws.com"
+            }
+            if (env.TIER == 'qa') {
+                        targetDomain = "https://wbeep-qa-website.s3-us-west-2.amazonaws.com"
+            }
           }
       }      
         success {
             mail to: 'gs-w_onhm@usgs.gov',
             subject: "${TIER} Success: ${currentBuild.fullDisplayName}",
             body: "Pipeline finished successfully \'${env.BUILD_URL}\'\n\n" +
             "Test result summary:\n" +
-            "https://wbeep-${TIER}-website.s3-us-west-2.amazonaws.com/estimated-availability/date/test_results/order_of_magnitude_test_${env.DATE}.html"
+            "${targetDomain}/estimated-availability/date/test_results/order_of_magnitude_test_${env.DATE}.html"
```

Comment:
I think this works but you can also just set `targetDomain` to "https://whatever/estimated-availability/date/test_results/order_of_magnitude_test_${env.DATE}.html" in lines 97, 100 and 103. That way you don't have to repeat the rest of that string down here. Or set env.URLSUFFIX = "/estimated-availability/date/test_results/order_of_magnitude_test_${env.DATE}.html" after line 92, before the if blocks and use that variable to append after targetDomain (`"${targetDomain}${env.URLSUFFIX}"`)

Reply:
But yeah, overall this looks fine